Unbound.cs

```diff
@@ -160,6 +178,16 @@ void Update()
         
         void OnGUI()
         {
+            if (GUILayout.Button("Test Modal"))
```

Comment:
Consider providing this button only in Debug builds? I don't think it doesn't provide much for users.

Reply:
Oh, good catch. The test button should be removed